recipe/meta.yaml

```diff
@@ -2,20 +2,26 @@ package:
     name: mingwpy
     version: 0.1.0
 
-source:                                                                                                # [win]
+source:
```

Comment:
It got clobbered in the merge. Probably due to the excess spaces here and my bad eyes.